src/patternfly/layouts/Flex/flex-order.scss

```diff
@@ -0,0 +1,25 @@
+$pf-l-flex--order--limit: $pf-global--order--limit;
```

Comment:
It seems like `pf-m-order-[0-12]` here is a bit arbitrary, since it's not based off of a 12 column/row layout. I can see adding it for consistency to match the grid layout. Just thinking forward, how might someone set `order: 42`? Would that have any impact on how we build this today? I don't think we would build out classnames for that.

One approach could be to have a fallback variable system at different breakpoints where we read `--order{-[breakpoint]}: n`, and we pass that directly to an `order` property at that breakpoint. That would work in place of the `pf-m-order` classes, and works for any `order` value. And the react component could take like `breakpoints={'md': 1, 'lg': -1}` and create `setProperty('--order-md', '1').setProperty('--order-lg', '-1')` (or however they do that).

Here's a demo of how that might work https://codepen.io/mcoker/pen/GRoeLJo